zap/src/main/java/org/zaproxy/zap/extension/api/API.java

```diff
@@ -929,6 +931,48 @@ public void removeCallBackUrls(ApiImplementor impl) {
         this.callBacks.values().removeIf(impl::equals);
     }
 
+    private String getCallbackKey(ApiImplementor impl, String site) {
+        return impl.getClass().getCanonicalName() + ":" + site;
```

Comment:
Might be better to use the API prefix rather than the class name, more stable (e.g. if the class is moved/renamed in a refactoring the key needs to be migrated).